egs/wsj/s5/utils/data/fix_subsegment_feats.pl

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env perl
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0.
+
+use warnings;
+
+# This script modifies the feats ranges and ensures that they don't 
+# exceed the max number of frames supplied in utt2max_frames.
```

Comment:
this documentation is not really sufficient.  a "feats range" is not something most readers will be familiar with, you need to give an example; and you don't state that this script reads from the stdin or give any example of what utt2max-frames might look like or what stdin might look like, and how this script will change the stdin before it prints it out.